src/main/kotlin/com/theworkshopvlc/chupapuntes/categories/CategoriesController.kt

```diff
@@ -2,18 +2,27 @@ package com.theworkshopvlc.chupapuntes.categories
 
 import com.theworkshopvlc.chupapuntes.categories.domain.Category
 import com.theworkshopvlc.chupapuntes.categories.domain.usecases.GetAllCategories
+import com.theworkshopvlc.chupapuntes.categories.domain.usecases.GetByTitle
 import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/categories")
-class CategoriesController(private val getAllCategories: GetAllCategories) {
+class CategoriesController(private val getAllCategories: GetAllCategories,
+                           private val getByTitle: GetByTitle) {
 
   @GetMapping("/index")
   @PreAuthorize("hasAnyAuthority('USER')")
   fun getAll(): List<Category> {
     return getAllCategories.execute()
   }
+
+  @GetMapping("/getByTitle")
```

Comment:
I would rename this route to `/title`

Reply:
You are right, better use nouns than verbs

Reply:
Refactor to the base endpoint and wait for title param https://github.com/theworkshopvlc/Chupapuntes/commit/4bad45e90d301bb15fe2091dea1a73f8299cb15d